examples/scripts/srpo/srpo_eval.py

```diff
@@ -0,0 +1,305 @@
+# flake8: noqa
```

Comment:
This is the eval that runs a more limited version of the paper eval with AlpacaFarm. Not entirely sure we want this in here but I left it for now.

Reply:
Thanks, that's very useful. For the moment evaluation is out of the scope of trl, but we are considering adding it soon. This script will be useful. We'll get it out of git hisyory in due course. Can you remove it in the meantime?